pysindy/feature_library/base.py

```diff
@@ -487,32 +509,11 @@ def get_feature_names(self, input_features=None):
         -------
         output_feature_names : list of string, length n_output_features
         """
-        feature_names = list()
-        for i in range(len(self.libraries)):
-            lib_i = self.libraries[i]
-            if input_features is None:
-                input_features_i = [
-                    "x%d" % k for k in _unique(self.inputs_per_library[i])
-                ]
-            else:
-                input_features_i = np.asarray(input_features)[
-                    _unique(self.inputs_per_library[i])
-                ].tolist()
-            lib_i_feat_names = lib_i.get_feature_names(input_features_i)
-            for j in range(i + 1, len(self.libraries)):
-                lib_j = self.libraries[j]
-                if input_features is None:
-                    input_features_j = [
-                        "x%d" % k for k in _unique(self.inputs_per_library[j])
-                    ]
-                else:
-                    input_features_j = np.asarray(input_features)[
-                        _unique(self.inputs_per_library[j])
-                    ].tolist()
-                lib_j_feat_names = lib_j.get_feature_names(input_features_j)
-                feature_names += self._name_combinations(
-                    lib_i_feat_names, lib_j_feat_names
-                )
+        feature_names = self._feature_names_at(0, input_features)
+        for i in range(1, len(self.libraries)):
+            lib_i_feat_names = self._feature_names_at(i, input_features)
```

Comment:
same as for library 0, we don't need `self._feature_names_at()` since we no longer have to worry about `input_features` ever being `None`